modules/nw-installing-external-dns-operator-cli.adoc

```diff
@@ -0,0 +1,160 @@
++
+.Example `namespace.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: external-dns-operator
+----
+
+.. Create the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file that defines the `OperatorGroup` object:
++
+.Example `operatorgroup.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  upgradeStrategy: Default
+  targetNamespaces:
+  - external-dns-operator
+----
+
+.. Create the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f operatorgroup.yaml
+----
+
+. Create a `Subscription` object:
+
+.. Create a YAML file that defines the `Subscription` object:
++
+.Example `subscription.yaml` file
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: external-dns-operator
+  namespace: external-dns-operator
+spec:
+  channel: stable-v1
+  installPlanApproval: Automatic
+  name: external-dns-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+----
+
+.. Create the `Subscription` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f subscription.yaml
+----
+
+.Verification
+
+. Get the name of the install plan from the subscription:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator \
+    get subscription external-dns-operator \
+    --template='{{.status.installplan.name}}{{"\n"}}'
+----
+
+. Check the status of the install plan:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator \
+    get ip <install_plan_name> \
+    --template='{{.status.phase}}{{"\n"}}'
+----
++
+.Required output
+[source,terminal]
+----
+Complete
+----
+
+. Check the status of the `external-dns-operator` pod is running:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get pod
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                                     READY   STATUS    RESTARTS   AGE
+external-dns-operator-5584585fd7-5lwqm   2/2     Running   0          11m
+----
+
+. Confirm the subscription is `Red Hat Operators` catalog source:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get subscription
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                    PACKAGE                 SOURCE             CHANNEL
+external-dns-operator   external-dns-operator   redhat-operators   stable-v1
+----
+
+. Check the `external-dns-operator` version:
++
+[source,terminal]
+----
+$ oc -n external-dns-operator get csv
+----
+
++
+.Example output
+[source,terminal]
+----
+NAME                           DISPLAY                VERSION   REPLACES   PHASE
+external-dns-operator.v1.2.0   ExternalDNS Operator   1.2.0                Succeeded
+----
```

Comment:
```suggestion
----
NAME                           DISPLAY                VERSION   REPLACES   PHASE
external-dns-operator.v<1.y.z>   ExternalDNS Operator   <1.y.z>                Succeeded
----
```

- You can add a call out in the output and something similar to:

 > <1> The `<1.y.z>` variable specifies      the External DNS Operator version that you installed, for example, `1.2.0`. Replace `<1.y.z>` with your External DNs Operator version.

- If you go with this suggestion, you can confirm with SME if `1.y.z` is appropriate. (Like in OCP we have 4.y.z ) [micro release guideline](https://redhat-documentation.github.io/supplementary-style-guide/#micro-release)

Reply:
Thanks, I'll take a stab at it.